src/org/usfirst/frc4904/robot/commands/IdleMotors.java

```diff
@@ -0,0 +1,46 @@
+package org.usfirst.frc4904.robot.commands;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+import edu.wpi.first.wpilibj.command.Subsystem;
+import org.usfirst.frc4904.standard.commands.motor.MotorIdle;
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+
+/**
+ * Idles motors of a subsystem.
+ * Used as a default command
+ *
+ */
+public class IdleMotors extends CommandGroup{
+
+	/**
+	 * Run MotorIdle (from WPILib) in parallel on every motor
+	 * from a list of motors with unrestricted size.
+	 *
+	 * @param subsystem
+	 * @param motors
+	 */
+	public IdleMotors(Subsystem subsystem, Motor[] motors) {
+		super("IdleMotors");
+		requires(subsystem);
+		for (int i=0; i<motors.length; i++) {
+			requires(motors[i]);
+			addParallel(new MotorIdle(motors[i]));
+		}
+	}
+
+	/**
+	 * Run MotorIdle (from WPILib) in parallel on each of 2 motors.
+	 *
+	 * @param subsystem
+	 * @param motor1
+	 * @param motor2
+	 */
+	public IdleMotors(Subsystem subsystem, Motor motor1, Motor motor2) {
```

Comment:
Could you make this just an extension of the earlier declaration, so you don't need to change things in both functions?